docs/troubleshooting/troubleshooting.mdx

```diff
@@ -0,0 +1,58 @@
+---
+title: Troubleshoot
+image: "/docs/images/docs-meta-cards/troubleshoot-card.png"
+displayed_sidebar: docs
+description: "Troubleshooting with Web3Auth | Documentation - Web3Auth"
+---
+
+import Tiles from "@theme/Tiles";
+
+Web3Auth offers multiple SDK support as well as a thriving community of developers discussing their implementations. Troubleshooting topics can help
```

Comment:
Web3Auth offers SDKs for various libraries across multiple platforms and devices. Thanks to our vibrant community of developers and their various implementations, we have collated some of the top asked questions as guides to help you quickly understand and figure out some of the edge cases in the implementations.

If you're still facing errors, you can always ask for help in the following channels:

- Have a look at our [GitHub Discussions](https://github.com/Web3Auth/Web3Auth/discussions?discussions_q=sort%3Atop) to see if anyone has any questions or issues you might be having.
- Join our [Discord](https://discord.gg/web3auth) to join our community and get private integration support or help with your integration.